.github/workflows/release-draft.yml

```
@@ -0,0 +1,19 @@
+name: Create/update draft release
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  deploy:
+    name: deploy
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      # Drafts your next Release notes as Pull Requests are merged
+      - uses: release-drafter/release-drafter@v5
+        with:
+          config-name: draft_release.yml
+        env:
+          GITHUB_TOKEN: ${{ secrets.BW_GITHUB_TOKEN }}
```

I don't know if this is actually required and would suggest we attempt without it for the moment.
```suggestion
        env:
          GITHUB_TOKEN: ${{ secrets.BW_GITHUB_TOKEN }}
```